### DISCUSSION TITLE
EL6 Feedback

### DISCUSSION THREAD

**Discussion by siwelwerd** [TeamBasedInquiryLearning/precalculus#222] [Show and tell]
https://teambasedinquirylearning.github.io/precalculus/2024/EL6.html

Author: @tdegeorge 
Reviewer: @AbbyANoble

**Comment by AbbyANoble** [comment]
It feels a little weird solving logarithmic ones first. Especially since the motivation for solving them is to make them exponential first.

**Reply by tdegeorge** [reply]
When Kathy and made the outline, we put the logs first because we intended on using that as motivation for solving exponentials later (i.e., the one with the like bases). But if we don't like that way, I can switch it around so that the exponentials come first.

**Reply by tdegeorge** [reply]
Ok, leaving this as we talked about this today in our meeting.

**Comment by AbbyANoble** [comment]
I know we all say "log" instead of "logarithm" in practice but it felt a little jarring to start shortening it in the writing. Especially when referencing the "quotient property of logs" instead of "quotient property of logarithms" in [Activity 5.6.5(a)](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/EL6.html#solving-log-equations-with-logs-on-both-sides-2).

Maybe a style thing we would want to decide.

**Reply by tdegeorge** [reply]
I personally don't have a preference....I'll do whatever the group wants

**Comment by AbbyANoble** [comment]
In [Activity 5.6.2](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/EL6.html#solve-by-converting), I think it would be better to frame it as these being simpler because a quick rewrite makes them easy to see the answer rather than appealing to not "wanting" to use a calculator. Maybe just without your first sentence?

**Comment by AbbyANoble** [comment]
[Activity 5.6.5(c)](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/EL6.html#solving-log-equations-with-logs-on-both-sides-4)  On part c I think you meant 2x-1=0 to match the original.

**Comment by AbbyANoble** [comment]
[Activity 5.6.5(d)](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/EL6.html#solving-log-equations-with-logs-on-both-sides-5)  - answer in part c, not d.

**Comment by AbbyANoble** [comment]
[Activity 5.6.9](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/EL6.html#solving-exponential-equations-by-converting) - Before converting, it might be nice to motivate why we can't just solve this already. If we solve logarithmic ones by making them exponential, why are we changing this exponential to a logarithmic?

Maybe something like "There is no whole number value we can raise 2 to to get 48. What two whole numbers must x be between?" (5 and 6). 

We'll use logarithms to isolate the variable in the exponent. (Insert your part a.)

Some comment on how this is the exact answer. (Which I do make my students find!)

Then ask to get an approximation. Not sure that they need the change of base though unless the calculator doesn't have the right buttons.

Then I'd have x \approx the answer choices instead of just the numbers to point out that it is indeed an approximate answer.

**Comment by AbbyANoble** [comment]
[Remark 5.6.10](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/EL6.html#EL6-3-11) - I'd be more specific again here. That logarithms help up get the variable out of the exponent.

**Comment by AbbyANoble** [comment]
[Activity 5.6.11](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/EL6.html#solving-by-like-bases) - Taking the log of both sides here feels out of place. I know that you are motivating that we are really just solving with like bases and we can just put the exponents equal to each other. But since solving by taking the log of both sides hasn't been introduced anywhere (nor do we have justification that it's valid algebraically) it feels like going around the world to get across the street. I think this is one of the places where students do buy in that if the bases match, so must the exponents.

**Reply by tdegeorge** [reply]
Left this activity after our discussion. :)

**Comment by AbbyANoble** [comment]
[Activity 5.6.14](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/EL6.html#solving-like-bases-scaffolded) - I might say something about that if the bases aren't the same, but we can rewrite them using properties of exponents, we can use the one-to-one property of exponentials. It feels like it could be framed better than saying it's not always easy.

**Reply by AbbyANoble** [reply]
Do we want them to solve these too? Or just rewrite?

**Reply by tdegeorge** [reply]
I was thinking just rewrite for this activity and then have them solve for the next one?

**Comment by AbbyANoble** [comment]
There aren't examples on how to solve exponentials when the bases don't match and can't be rewritten with the same base (taking log of both sides). But there are practice problems where they need that method -- see  [Activity 5.6.15(d)](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/EL6.html#solving-both-practice-5) and [Activity 5.6.16(b)](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/EL6.html#solving-both-practice-2-3).

**Reply by tdegeorge** [reply]
Good point! I'll add those in.

**Comment by AbbyANoble** [comment]
[Activity 5.6.15(d)](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/EL6.html#solving-both-practice-5) - Can you include exact answer too?

**Comment by AbbyANoble** [comment]
[Activity 5.6.15](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/EL6.html#solving-both-practice) and [Activity 5.6.16](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/EL6.html#solving-both-practice-2) have the same instructions. Not sure why they are split up? Could you do one set of exponential practice and one set of logarithms?

**Reply by tdegeorge** [reply]
I think the idea was that the first activity would provide MC to help students if they needed it and the second activity does not.
